tests/pact/signatures.repl

```diff
@@ -12,6 +12,12 @@
   (defconst TEST1-C1 0)
   (defconst TEST1-C2 "const 1")
   (defconst TEST1-C3 2.0)
+
+  (defcap MGD_CAP:bool () @managed MGD_CAP_mgr)
+  (defun MGD_CAP_mgr:object (mgd:object req:object) "Mgr for CAP")
```

Comment:
every managed cap will have the same shape (managed + requested), so it doesn't make much sense to define a management defun signature. If you state a cap signature is managed in an interface, then we should insert a corresponding signature into the interface's refs.

Reply:
True ... but this would require making `@managed` polymorphic. Today it needs to refer to a `TDef`, so it's hard to do that without including that in the interface.